app/pb_kits/playbook/packs/site_styles/_getting_started.scss

```
@@ -0,0 +1,18 @@
+@import "../../tokens/colors";
+@import "../../tokens/spacing";
+
+.getting_started {
+  display: flex;
+  justify-content: center;
+  padding: 32px 40px;
```

```suggestion
  padding: $space_lg $space_xl;
```
Why not use our tokens?

---

Missed this one, can definitely use spacing here.